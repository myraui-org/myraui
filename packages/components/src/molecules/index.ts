export * from './form-inputs';
export * from './toast';
export * from './theme-selector';
export * from './theme-switcher';
export * from './form';
