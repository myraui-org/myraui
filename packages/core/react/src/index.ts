'use client';
export * from '@myraui/components';
// export * from '@myraui/hooks';
export * from '@myraui/react-utils';
export * from '@myraui/shared-utils';
// export * from '@myraui/theme';
