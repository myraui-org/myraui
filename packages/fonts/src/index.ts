export { eudoxusSans } from './fonts/eudoxus-sans'
