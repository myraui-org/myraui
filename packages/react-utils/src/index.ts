export * from './context';
export * from './theme';
export * from './types';
