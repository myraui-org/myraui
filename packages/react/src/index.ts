'use client';
export * from './components';
export * from '@myraui-org/react-utils';
export * from '@myraui-org/shared-utils';
