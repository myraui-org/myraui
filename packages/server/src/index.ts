export * from '@myraui-org/fonts';
