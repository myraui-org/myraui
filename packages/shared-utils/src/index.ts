export * from './types';
export * from './functions';
export * from './objects';
export * from './assertion';
export * from './clsx';
export * from './errors';
export * from './strings';
