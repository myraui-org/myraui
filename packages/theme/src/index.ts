export * from './components';
export * from './plugin';
export * from './utilities';
