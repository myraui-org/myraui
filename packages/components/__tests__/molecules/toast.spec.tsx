import * as React from 'react';
import { render } from '@testing-library/react';
import { toast, Toaster } from '../../src';

describe('Toast', () => {
  it('should render correctly', () => {
    toast({ title: 'Something went wrong' });

    const wrapper = render(<Toaster />);
    expect(wrapper.container).toMatchSnapshot();

    expect(() => wrapper.unmount()).not.toThrow();
  });
});
