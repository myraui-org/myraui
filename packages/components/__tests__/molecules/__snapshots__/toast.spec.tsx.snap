// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast should render correctly 1`] = `
<div>
  <div
    aria-label="Notifications (F8)"
    role="region"
    style=""
    tabindex="-1"
  >
    <span
      aria-hidden="true"
      style="position: fixed; border: 0px; width: 1px; height: 1px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; word-wrap: normal;"
      tabindex="0"
    />
    <ol
      class="fixed top-0 z-[100] flex max-h-screen w-full flex-col-reverse p-4 sm:bottom-0 sm:right-0 sm:top-auto sm:flex-col md:max-w-[420px]"
      tabindex="-1"
    >
      <li
        aria-atomic="true"
        aria-live="off"
        class="group pointer-events-auto relative flex w-full items-center justify-between space-x-2 overflow-hidden rounded-md p-4 pr-6 shadow-lg data-[swipe=cancel]:translate-x-0 data-[swipe=end]:translate-x-[var(--radix-toast-swipe-end-x)] data-[swipe=move]:translate-x-[var(--radix-toast-swipe-move-x)] data-[swipe=move]:transition-none border border-default-100 bg-default text-default-foreground transition-all data-[state=open]:animate-in data-[state=closed]:animate-out data-[swipe=end]:animate-out data-[state=closed]:fade-out-80 data-[state=closed]:slide-out-to-right-full data-[state=open]:slide-in-from-top-full data-[state=open]:sm:slide-in-from-bottom-full"
        data-radix-collection-item=""
        data-state="open"
        data-swipe-direction="right"
        role="status"
        style="user-select: none;"
        tabindex="0"
      >
        <div
          class="grid gap-1"
        >
          <div
            class="text-sm font-semibold [&+div]:text-xs"
          >
            Something went wrong
          </div>
        </div>
        <button
          class="absolute right-1 top-1 rounded-md p-1 text-foreground/50 opacity-0 transition-opacity hover:text-foreground focus:opacity-100 focus:outline-none focus:ring-1 group-hover:opacity-100 group-[.danger]:text-danger-300 group-[.danger]:hover:text-danger-50 group-[.danger]:focus:ring-danger-400 group-[.danger]:focus:ring-offset-danger-600 group-[.success]:text-success-300 group-[.success]:hover:text-success-50 group-[.success]:focus:ring-success-400 group-[.success]:focus:ring-offset-success-600 group-[.warning]:text-warning-300 group-[.warning]:hover:text-warning-50 group-[.warning]:focus:ring-warning-400 group-[.warning]:focus:ring-offset-warning-600 group-[.primary]:text-primary-300 group-[.primary]:hover:text-primary-50 group-[.primary]:focus:ring-primary-400 group-[.primary]:focus:ring-offset-primary-600 group-[.secondary]:text-secondary-300 group-[.secondary]:hover:text-secondary-50 group-[.secondary]:focus:ring-secondary-400 group-[.secondary]:focus:ring-offset-secondary-600"
          data-radix-toast-announce-exclude=""
          toast-close=""
          type="button"
        >
          <svg
            class="tabler-icon tabler-icon-x h-4 w-4"
            fill="none"
            height="24"
            stroke="currentColor"
            stroke-linecap="round"
            stroke-linejoin="round"
            stroke-width="2"
            viewBox="0 0 24 24"
            width="24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M18 6l-12 12"
            />
            <path
              d="M6 6l12 12"
            />
          </svg>
        </button>
      </li>
    </ol>
    <span
      aria-hidden="true"
      style="position: fixed; border: 0px; width: 1px; height: 1px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; word-wrap: normal;"
      tabindex="0"
    />
  </div>
</div>
`;
